### PR TITLE
Code for the issue created.

### PR DESCRIPTION
**Pull request by codingwithveer18**
## 🛠️ Issue #1007 

# Description

Code for my project i.e regarding Mental Health Awareness Initiative.

## Type of change

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

-I have performed a self-review of my own code
-I have commented my code, particularly in hard-to-understand areas
-I have made corresponding changes to the documentation
-My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
https://soulboundconfessions.netlify.app/
Screenshot 2023-06-29 003643
Screenshot 2023-06-29 003701
Screenshot 2023-06-29 003713
Screenshot 2023-06-29 003724
Screenshot 2023-06-29 003733
Screenshot 2023-06-29 003743

### CODE REVIEW

**Review by harshit-paneri — CHANGES_REQUESTED** [review]
File path is wrong
Your file path : /Soulbound_Confession/
correct file path  : /Projects/Soulbound_Confession/

**Comment by codingwithveer18** [comment]
Changes Done . Please Review

**Comment by harshit-paneri** [comment]
Did you make figma

**Comment by codingwithveer18** [comment]
> Did you make figma

Yes , for the app ui , i made a figma prototype

**Comment by codingwithveer18** [comment]
@harshit-paneri  
Sir , Are there any more changes that to be made ?

**Comment by codingwithveer18** [comment]
That app.html file is just for the prototype.
main file is index one.
Is there anything to be checked sir @harshit-paneri

**Comment by codingwithveer18** [comment]
Also, my issue is not showing in Front-End Projects (under projects section on github).

**Review by codingwithveer18 — COMMENTED** [review]
minor changes

**Review by TusharKesarwani — CHANGES_REQUESTED** [review]
Revert changes in index.html file

**Comment by codingwithveer18** [comment]
> Revert changes in index.html file

Delete the path of the deleted file ,
Which was a figma prototype .
So do i have to add that again ?

**Comment by TusharKesarwani** [comment]
> > Revert changes in index.html file
> 
> Delete the path of the deleted file , Which was a figma prototype . So do i have to add that again ?

You should revert changes made in the main index.html file

**Comment by codingwithveer18** [comment]
> > > Revert changes in index.html file
> > 
> > Delete the path of the deleted file , Which was a figma prototype . So do i have to add that again ?
> 
> You should revert changes made in the main index.html file

Done sir

File: index.html
Comment (TusharKesarwani) [inline index.html:1]: Revert this file

Comment (codingwithveer18) [reply index.html:1]: Now okay sir ?

**Comment by codingwithveer18** [comment]
Changes Done . Please look into that

**Review by TusharKesarwani — CHANGES_REQUESTED** [review]
Do not add bootstrap files in folder
Connect them through link

**Comment by codingwithveer18** [comment]
Please look into this

**Review by TusharKesarwani — CHANGES_REQUESTED** [review]
Remove font awesome files and connect them remotely like Bootstrap

**Comment by codingwithveer18** [comment]
> Remove font awesome files and connect them remotely like Bootstrap

I'm unaware about this. I am not able to convert them remotely.

**Comment by TusharKesarwani** [comment]
> > Remove font awesome files and connect them remotely like Bootstrap
> 
> I'm unaware about this. I am not able to convert them remotely.

Go to their website or search it on youtube

**Comment by codingwithveer18** [comment]
> > > Remove font awesome files and connect them remotely like Bootstrap
> > 
> > 
> > I'm unaware about this. I am not able to convert them remotely.
> 
> Go to their website or search it on youtube

Done Sir

**Review by TusharKesarwani — APPROVED** [review]

**Review by TusharKesarwani — CHANGES_REQUESTED** [review]
Screenshot
Make it responsive

**Review by TusharKesarwani — CHANGES_REQUESTED** [review]
Screenshot (1)

Make it responsive

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**